tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -206,36 +258,53 @@ Create VCH - Existing VCH name
     Log To Console  Installer completed successfully: %{VCH-NAME}
 
     ${output}=  Run  bin/vic-machine-linux create --name=%{VCH-NAME} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} --bridge-network=%{BRIDGE_NETWORK} --public-network=%{PUBLIC_NETWORK} --compute-resource=%{TEST_RESOURCE} ${vicmachinetls}
-    Should Contain  ${output}  already exists. Please delete it before reinstalling.
+    ${vm-folder-path}=  Run  govc ls | grep vm
+    Should Contain  ${output}  A VCH with the name \\"%{VCH-NAME}\\" already exists. Please delete it or choose a different VCH name before attempting reinstalling
 
     Cleanup VIC Appliance On Test Server
 
-Create VCH - Existing VM name
-    ${status}=  Get State Of Github Issue  7499
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 6-04-Create-Basic.robot needs to be updated now that Issue #7499 has been resolved
-    # Set Test Environment Variables
-    # Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
-    # Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+Create VCH - Existing VM Name
```

Comment:
I'd probably leave this test as it was and create a new test that only runs on VC and checks the various folder paths...

Reply:
See line 291, there is a separate text case that is the same as this one but checks folders :)

Reply:
We do need specific logic for this test based on ESX vs. VC to maintain the way it worked before. Since we must still make a dummy vm for the spirit of this test.